src/components/mapComponents/mapPageComponents/mapPage/index.tsx

```diff
@@ -1,43 +1,32 @@
 import React from 'react';
 import MapSidebar from '../mapSidebar';
-import PageLayout from '../../pageLayout';
+import PageLayout from '../../../pageLayout';
 import { Layout } from 'antd';
-import { MainContent } from '../../themedComponents';
-import MapContent from '../mapContent';
-import { MapGeoDataReducerState, MapViews } from '../ducks/types';
-import MapLegend from '../mapLegend';
-import { WindowTypes } from '../../windowDimensions';
+import { MainContent } from '../../../themedComponents';
+import { MapViews } from '../../ducks/types';
+import MapLegend from '../../mapLegend';
+import { WindowTypes } from '../../../windowDimensions';
 
 interface MapPageProps {
+  readonly mapContent: JSX.Element;
```

Comment:
❓ Does this give you any console errors? I know I've had some trouble passing JSX as props like this

Reply:
I'm not seeing any–what kind of errors were they?